src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderProgramProfile.kt

```diff
@@ -39,7 +39,7 @@ data class OffenderProgramProfile(
   var program: ProgramService,
 
   @Column(name = "OFFENDER_START_DATE")
-  var startDate: LocalDate,
+  var startDate: LocalDate?,
```

Comment:
We used to treat this as non-nullable because it was never null for the allocations we're interested in. As we now use this column to check if an allocation was ever activated we have to cater for null values - e.g. those on the waiting list.